app/grandchallenge/evaluation/models.py

```diff
@@ -616,6 +641,15 @@ class Meta:
             ("configure_algorithm_phase", "Configure Algorithm Phase"),
         )
 
+    def get_selectable_gpu_type_choices_evaluation(self):
+        choices = {GPUTypeChoices.NO_GPU, GPUTypeChoices.T4}
+        if self.selectable_gpu_type_choices_evaluation and isinstance(
+            self.selectable_gpu_type_choices_evaluation, list
+        ):
+            for choice in self.selectable_gpu_type_choices_evaluation:
+                choices.add(GPUTypeChoices[choice])
+        return choices
```

Comment:
Hmm, good question. Maybe rather than combining sets they are two independent lists?

Reply:
Looking at this further I think that if we define it as the default we then only have one list, and can take the value of `selectable_gpu_type_choices_evaluation` directly rather than having to combine anything.